client/signup/processing-screen/index.tsx

```diff
@@ -135,10 +135,14 @@ export default function ProcessingScreen( props: ProcessingScreenProps ) {
 				'is-force-centered': shouldShowNewSpinner && totalSteps === 0,
 			} ) }
 		>
-			<h1 className="processing-screen__progress-step">{ steps.current[ currentStep ]?.title }</h1>
-			{ shouldShowNewSpinner && <LoadingEllipsis /> }
+			{ shouldShowNewSpinner && (
+				<Loading title={ steps.current[ currentStep ]?.title } progress={ progress } />
```

Comment:
I believe progress is passed here as a value 0 < progress < 1, while the Loading component goes from 0 to 100.